### PR TITLE
Add Internet Access Policy

### PR DESCRIPTION
**Pull request by sindresorhus**
See https://obdev.at/iap/index.html

This makes it clearer for people using Little Snitch (most popular firewall) why we connect to the server and it makes it less likely they will deny it.

### CODE REVIEW

File: Gifski/InternetAccessPolicy.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"ApplicationDescription": "Gifski converts videos to high-quality GIFs.",
+	"DeveloperName": "Sindre Sorhus",
+	"Website": "https://sindresorhus.com/dato",
+	"Connections" : [
+		{
+			"Host" : "*.crashlytics.com",
+			"NetworkProtocol" : "TCP",
+			"Port" : "443",
+			"Purpose" : "Gifski sends crash reports to this server.",
+			"DenyConsequences" : "If you deny this connection, crash reports will not be sent and bugs might stay unfixed."
```
Comment (sindresorhus) [inline Gifski/InternetAccessPolicy.json]: I'm not 100% happy about this sentence.

Comment (sunshinejr) [inline Gifski/InternetAccessPolicy.json]:
```suggestion
			"DenyConsequences" : "If you deny this connection, we won't be able to receive your crash reports. And if we don't know about them, we can't fix them."
```

not 100% happy with this one too but maybe you'll find it helpful 😅

Comment (sindresorhus) [reply Gifski/InternetAccessPolicy.json]:
```diff
-If you deny this connection, we won't be able to receive your crash reports. And if we don't know about them, we can't fix them.
+If you deny this connection, the developers of Gifski won't be notified of any crashes and they can't fix what they don't know about.
```

?

I feel like "your" implies that the user is actually sending the crash reports explicitly, which is not the case. I also think "we" is not clear enough in this context.

Comment (sunshinejr) [reply Gifski/InternetAccessPolicy.json]: @sindresorhus good point, looks better 👍

**Pull request merged**